ibis/backends/conftest.py

```diff
@@ -4,13 +4,15 @@
 import importlib
 import importlib.metadata
 import itertools
+import operator
 from functools import cache
 from pathlib import Path
 from typing import TYPE_CHECKING, Any
 
 import _pytest
 import numpy as np
 import pandas as pd
+import pkg_resources
```

Comment:
Can you use `importlib.metadata` instead? `pkg_resources` is kind of awful.

Reply:
Sure, updating.